meta-lxatac-bsp/recipes-core/images/emmc-boot-image/lxatac/genimage.config

```diff
@@ -18,7 +18,7 @@ image @IMAGE@fip {
     fip {
         fw-config = "stm32mp157c-lxa-tac-fw-config.dtb"
         hw-config = "stm32mp157c-lxa-tac-fw-config.dtb"
-        nt-fw = "barebox-stm32mp15xc-lxa-tac.img"
+        nt-fw = "barebox-stm32mp15xc-lxa-tac.stm32"
```

Comment:
The .stm32 image contains at 256 byte header that I suspect will be executed as code. The .img omitted that header.

Reply:
My bad. This was a result of me missing the fixup that did the `stm32` --> `img` transition.
The fixup from https://github.com/linux-automation/meta-lxatac/commit/e285dad4f06688c9df3f5b1d9720a7db5191542b is now back in and this change no longer required.